### DISCUSSION TITLE
different power on/off commands

### DISCUSSION THREAD

**Discussion by astlgit** [Geektoolkit/Dynaframe3#164] [Ideas]
would it be possible to use cec-hdmi commands instead of the vcgencmd for the powering on and off of the monitor? I'm using a Samsung tv for my display and it turns off fine but when I "SCREENON" the tv is in sleep/off mode and needs to be powered on by a remote. I've installed cec-hdmi on my rpi and confirmed that it works with both commands of on/off.

**Comment by qwksilver** [comment]
i assume that cec-hdmi is an apt get thing.
if you have the two commands already reply below and lay those out for us and we will look into it.

@Geektoolkit this might solve an issue i have with one system also.

**Reply by Geektoolkit** [reply]
I'll have to look into it. May have to add an option. My TVs didn't support CEC control last I tried

**Reply by astlgit** [reply]
https://pimylifeup.com/raspberrypi-hdmi-cec/
